src/cmds/hardware/ccid/ccid_cmd.c

```diff
@@ -166,8 +166,8 @@ static void handle_ccid_commands(struct usb_interface *udev) {
 		while (str != NULL) {
 			unsigned int c;
 
-			sscanf(str, "%x", &c);
-			cmd_buf[cmd_len++] = c;
+			if (sscanf(str, "%x", &c) == 1)
+				cmd_buf[cmd_len++] = c;
```

Comment:
Could you use braces even for a single line code

Reply:
Sure. Modified.